src/vizier/services/query_broker/query_broker_server.go

```diff
@@ -205,11 +209,22 @@ func main() {
 	}()
 	defer ptProxy.Close()
 
-	// Start cron script runner.
-	sr, err := scriptrunner.New(natsConn, csClient, vzServiceClient, viper.GetString("jwt_signing_key"))
+	kubeConfig, err := rest.InClusterConfig()
```

Comment:
And we would just pass in the namespace string? Or would we include the `viper.GetString("pod_namespace")`?

Reply:
If it were entirely up to me, I would lean towards passing in the namespace string just to avoid propagating knowledge that there is a config option named "pod_namespace" to more than one module.